src/migrations/20210120220221-add-collaborators-to-activity-reports.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.createTable('ActivityReportCollaborators', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      activityReportId: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: {
+            tableName: 'ActivityReports',
+          },
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'CASCADE',
```

Comment:
I always mess these up. Does this delete the ActivityReport when a collaborator is removed, or delete the collaborator automatically when an ActivityReport is deleted? (Same question for user)

Reply:
I wasn't 100% sure either:

> CASCADE specifies that when a referenced row is deleted, row(s) referencing it should be automatically deleted as well.

from https://www.postgresql.org/docs/9.5/ddl-constraints.html